src/sources/exec/mod.rs

```diff
@@ -264,10 +259,6 @@ impl SourceConfig for ExecConfig {
         vec![Output::default(self.decoding.output_type())]
     }
 
-    fn source_type(&self) -> &'static str {
-        EXEC
```

Comment:
Can we delete the `const` as well?

Reply:
Hmm, it does use it on line 578 but I could replace that with `ExecConfig::get_component_name()`.

Good eye.